R/RunDiagnostics.R

```diff
@@ -710,6 +715,37 @@ executeDiagnostics <- function(cohortDefinitionSet,
       }
     )
   }
+  
+  # Defines variables and checks version of external concept counts table -----
+  if (useExternalConceptCountsTable == FALSE) {
+    conceptCountsTableIsTemp <- TRUE
+    if (conceptCountsTable != "#concept_counts") {
+      conceptCountsTable <- "#concept_counts"
+    }
+  } else {
+    if (conceptCountsTable == "#concept_counts") {
+      stop("Temporary conceptCountsTable name. Please provide a valid external ConceptCountsTable name")
+    }
+    conceptCountsTableIsTemp <- FALSE
```

Comment:
This flag variable is not needed as it is logically equivalent to `useExternalConceptCountsTable`